cmd/opera/launcher/export.go

```diff
@@ -138,7 +138,11 @@ func exportEvmKeys(ctx *cli.Context) error {
 	keysDB := batched.Wrap(keysDB_)
 	defer keysDB.Close()
 
-	it := gdb.EvmStore().EvmDb.NewIterator(nil, nil)
+	evmDb := gdb.EvmStore().EvmDb
+	if evmDb == nil {
+		panic("unknown EvmDb format")
+	}
```

Comment:
We might wrap these into a function in `evmstore.Store` to avoid duplicate code?

Reply:
Better I will add different error messages which explain each case.